src/sage/rings/polynomial/polynomial_quotient_ring.py

```diff
@@ -1260,12 +1260,15 @@ def polynomial_ring(self):
 
     cover_ring = polynomial_ring
 
-    def random_element(self, *args, **kwds):
+    def random_element(self, *args, degree=None, **kwds):
         """
         Return a random element of this quotient ring.
 
         INPUT:
 
+        - ``degree`` - Optional argument: either an integer for fixing the
+          degree, or a tuple of the minimum and maximum degree. By default the
+          degree is n - 1 with n the degree of the polynomial ring.
```

Comment:
To be explicit it might be worth it to state here that the degree is only fixed up to modulo calculation, so that the user can check the documentation when the query for a degree 20 element in a quotient ring with modulus of degree 5 returns and element of degree at most 4.

Reply:
Yeah, the way you wrote it now is good - it was just about the user not being confused when the output representative doesn't have the degree that was given as input.